packages/ember-principled-forms/addon/components/principled-form-field/component.ts

```diff
@@ -2,6 +2,8 @@ import Component from '@ember/component';
 import { assert } from '@ember/debug';
 import { action } from '@ember-decorators/object';
 
+// DISCUSS: Do we really need a lodash dependency?
+// DISCUSS: I also wonder if there is a way to design these packages without requiring true-myth
```

Comment:
Is there a reason to *avoid* using True Myth? It's a *tiny* dependency, and especially in our apps will usually be present anyway. More importantly, it's an under-the-hood dependency, so it doesn't surface to consumers.

Reply:
Nothing other than my gut instincts to minimize dependencies wherever possible in a library I want to ship to other people. I don't have a strong feeling; I was just wondering how difficult it would be to make `true-myth` an optional dependency.